src/server/game/Handlers/MovementHandler.cpp

```diff
@@ -325,11 +325,14 @@ void WorldSession::HandleMovementOpcodes(WorldPacket & recvData)
 
     recvData.rfinish();                         // prevent warnings spam
 
-    // pussywizard: typical check for incomming movement packets
-    if (!mover || !mover->IsInWorld() || mover->IsDuringRemoveFromWorld() || guid != mover->GetGUID())
+    // prevent tampered movement data
```

Comment:
Need check guild before `ReadMovementInfo(recvData, &movementInfo);` - #2426 [R322-R331](https://github.com/azerothcore/azerothcore-wotlk/pull/2426/files#diff-649b0ea06820eb06283d1ccc8c83efdbR322-R331)

Reply:
Done, thank you.